content/en/docs/started/installing-kubeflow.md

```diff
@@ -18,8 +18,11 @@ Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
 Anywhere you are running Kubernetes, you should be able to run Kubeflow.
 There are two primary ways to install Kubeflow:
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+- [What is Kubeflow?](#what-is-kubeflow)
+- [How to install Kubeflow?](#how-to-install-kubeflow)
+- [Packaged Distributions of Kubeflow](#packaged-distributions-of-kubeflow)
+- [Raw Kubeflow Manifests](#raw-kubeflow-manifests)
+- [Next steps](#next-steps)
```

Comment:
Why do we need this change ?

Reply:
My bad vscode expanded the menu via a plugin. Let me remove it.